README.md

```diff
@@ -1,55 +1,140 @@
+[[YouTube Video]](https://www.youtube.com/watch?v=BHZnA-CZeZY) [[Website]](https://amathislab.github.io/BUCTD/)
 
-- This work is accepted to ICCV 2023 🎉
 
-- This work was also presented at the 2023 [CV4Animals workshop at CVPR](https://www.cv4animals.com)
+- Sep 2023: We released the code :)
+- July 2023: This work is accepted to ICCV 2023 🎉
+- June 2023: BUCTD was also presented at the 2023 [CV4Animals workshop at CVPR](https://www.cv4animals.com)
+- June 2023: An earlier version can be found on [arxiv](https://arxiv.org/abs/2306.07879)
 
-![BUCTD](media/BUCTD_fig1.png)
-
-- Stand-alone code will be made available by ICCV (Oct 2023)!
+<img src="media/BUCTD_fig1.png" width="600">
 
 - This code will also be integrated in [DeepLabCut](https://github.com/DeepLabCut/DeepLabCut)!
 
-### Code Acknowledgements
+### Installation 
+
+We developed and tested our models under ```python=3.8.10, pytorch=1.8.0, cuda=11.1```. Other versions might be working as fine.
+
+<details>
+  <summary>Instructions</summary>
+   
+   1. Clone this repo, and we'll call the directory that you cloned as ${BUCTD_ROOT}.
+
+   ```sh
+   git clone https://github.com/amathislab/BUCTD.git
+   cd ${BUCTD_ROOT}
+   ```
+
+   2. Install Pytorch and torchvision
+
+   Follow the instruction on https://pytorch.org/get-started/locally/.
+   ```sh
+   # an example:
+   conda install pytorch==1.8.0 torchvision==0.9.0 cudatoolkit=11.1 -c pytorch -c conda-forge
+   ```
+
+   3. Install other needed packages
+   
+   ```sh
+   pip install -r requirements.txt
+   ```
+
+   4. Install [COCOAPI](https://github.com/cocodataset/cocoapi)
+   
+   ```sh
+   # COCOAPI=/path/to/clone/cocoapi
+   git clone https://github.com/cocodataset/cocoapi.git $COCOAPI
+   cd $COCOAPI/PythonAPI
+   # Install into global site-packages
+   make install
+   # Alternatively, if you do not have permissions or prefer
+   # not to install the COCO API into global site-packages
+   python setup.py install --user
+   ```
+   
+   5. Install [CrowdPoseAPI](https://github.com/Jeff-sjtu/CrowdPose) exactly in the same way as COCOAPI.
+
+   6. Install NMS
+   
+   ```sh
+   cd ${BUCTD_ROOT}/lib
+   make
+   ```
+   
+</details>
+
+
+### Training
+<details>
+  <summary>Instructions</summary>
+
+***Generative sampling***
+
+You can use the script: ```train_BUCTD_synthesis_noise.sh```.
+
+***Empirical sampling***
+
+You can match your own BU models by updating the scripts in [./data_preprocessing/](./data_preprocessing/). 
 
-We are grateful to the authors of [HRNet](https://github.com/HRNet/deep-high-resolution-net.pytorch) and [MIPNet](https://rawalkhirodkar.github.io/mipnet) as our code builds on their excellent work. 
+If you don't want to match your own BU models for training, we provide the training annotations. You can download the annotations [here](https://drive.google.com/drive/folders/17UnDWUtvcXrmuH90_wYrDUxYDOBkQ0C1?usp=share_link).
```

Comment:
BU needs defined

Reply:
Not fixed? @LucZot

Reply:
It is now defined in the sentence before